pkg/cmd/config.go

```diff
@@ -48,8 +48,9 @@ const (
 type printOpts struct {
 	FullConfig bool `longflag:"full" shortflag:"f"`
 
-	ClusterName       string `longflag:"cluster-name" shortflag:"n"`
-	KubernetesVersion string `longflag:"kubernetes-version" shortflag:"k"`
+	ClusterName         string `longflag:"cluster-name" shortflag:"n"`
+	KubernetesVersion   string `longflag:"kubernetes-version" shortflag:"k"`
+	ContainerLogMaxSize string `longflag:"container-log-max-size"`
```

Comment:
Very tiny nit: I would put this at the end, so variables are grouped by priority/importance.

Reply:
fixed